packages/client/src/runtime/core/engines/data-proxy/DataProxyEngine.ts

```diff
@@ -66,6 +66,7 @@ type DataProxyHeaders = {
   Authorization: string
   'X-capture-telemetry'?: string
   traceparent?: string
+  'Prisma-Engine-Hash': string
```

Comment:
```suggestion
  'X-Prisma-Engine-Hash': string
```

Because:

> Some web applications use custom headers to add comments or annotations to an HTTP message. The convention is to prefix the header name with X- to indicate that it is non-standard

http://www.httpwatch.com/HTTPgallery/headers/#:~:text=Some%20web%20applications%20use%20custom,that%20it%20is%20non%2Dstandard.

Reply:
>On June 2012, the deprecation of recommendation to use the "X-" prefix has become official as [RFC 6648](https://www.rfc-editor.org/rfc/rfc6648).
https://www.rfc-editor.org/rfc/rfc6648

`X-` is to be avoided since 2012, it was a recommendation before.

Reply:
But we could keep it for consistency if we have others `X-` headers I guess...